### PR TITLE
8191406: [hidpi] sun/java2d/SunGraphics2D/DrawImageBilinear.java test fails

### PR DESCRIPTION
**Pull request by jayathirthrao**
This test was failing because of timing or color difference issues.
Updated test to:

1) Use EDT for UI drawing instead of manual synchronisation
2) Use Robot.waitForIdle before screen capture
3) Increase color tolerance delta from 1 to 5
4) Use undecorated frame and explicitly set background color and move the frame to top
5) Cleanup to remove not needed flags and java options

After these changes test passes on all platforms in CI except Linux. For Linux it looks like product issue so i have raised : https://bugs.openjdk.org/browse/JDK-8297175

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8191406](https://bugs.openjdk.org/browse/JDK-8191406): [hidpi] sun/java2d/SunGraphics2D/DrawImageBilinear.java test fails

### Reviewers
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11201/head:pull/11201` \
`$ git checkout pull/11201`

Update a local copy of the PR: \
`$ git checkout pull/11201` \
`$ git pull https://git.openjdk.org/jdk pull/11201/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11201`

View PR using the GUI difftool: \
`$ git pr show -t 11201`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11201.diff">https://git.openjdk.org/jdk/pull/11201.diff</a>

</details>

### CODE REVIEW

**Comment by mrserb** [comment]
>Increase color tolerance delta from 1 to 5

I see that the tolerance is added to the test, previously it was 0. Why do we need it since we use quite simple RED color?

File: test/jdk/sun/java2d/SunGraphics2D/DrawImageBilinear.java [hunk 1]
```diff
@@ -156,63 +144,58 @@ private static void testRegion(BufferedImage bi,
...
-        skipOglTextureTest = false;
-        System.out.println("Skip OpenGL texture test: " + skipOglTextureTest);
-
-        DrawImageBilinear test = new DrawImageBilinear();
-        Frame frame = new Frame();
+    private static void createAndShowGUI() {
+        test = new DrawImageBilinear();
+        frame = new Frame();
         frame.add(test);
+        frame.setBackground(Color.WHITE);
```
Comment (mrserb) [inline test/jdk/sun/java2d/SunGraphics2D/DrawImageBilinear.java:151]: How the specific background color helps the test to pass, as far as I understand it should not affect the test, no?

Comment (jayathirthrao) [reply test/jdk/sun/java2d/SunGraphics2D/DrawImageBilinear.java:151]: I noticed in my local Ubuntu VM that sometimes the background was not pure white and had some delta. Thats why i explicitly added background color. But looks like it is not needed and test passes all the time in CI, so i have removed it.

**Comment by jayathirthrao** [comment]
> > Increase color tolerance delta from 1 to 5
> 
> I see that the tolerance is added to the test, previously it was 0. Why do we need it since we use quite simple RED color?

We have a tolerance of 1 for red color. We dont throw exception when red channel is 254.
But i think we dont need extra tolerance and it was added because of local machine reporting differences.
I have removed the extra tolerance and see that test passes in CI.

**Comment by jayathirthrao** [comment]
I am ignoring the previous comment. Even after translating, the comment looks to be not related to current PR.
@mrserb please take a look at updated PR.

File: test/jdk/sun/java2d/SunGraphics2D/DrawImageBilinear.java [hunk 2]
```diff
@@ -156,63 +131,57 @@ private static void testRegion(BufferedImage bi,
...
-                    test.wait();
-                } catch (InterruptedException e) {
-                    throw new RuntimeException("Failed: Interrupted");
-                }
-            }
+    public static void main(String[] args) throws Exception {
+        try {
+            EventQueue.invokeAndWait(() -> createAndShowGUI());
+        } catch (Exception ex) {
+            ex.printStackTrace();
```
[the remark this answers is not in the document]
Comment (jayathirthrao) [reply test/jdk/sun/java2d/SunGraphics2D/DrawImageBilinear.java]: Tested and updated.

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**